src/Components/Onboarding/Steps/Genes/SuggestedGenes.tsx

```diff
@@ -27,11 +27,13 @@ class SuggestedGenesContent extends React.Component<RelayProps, null> {
 const SuggestedGenesContainer = createFragmentContainer(
   SuggestedGenesContent,
   graphql`
-    fragment SuggestedGenesContent_suggested_genes on SuggestedGene @relay(plural: true) {
+    fragment SuggestedGenesContent_suggested_genes on Gene @relay(plural: true) {
       id
       name
       _id
-      image_url
+      image {
+        url
+      }
```

Comment:
This will probably lead to a merge error with work that @xtina-starr and @yuki24 have been doing.